packages/eslint-plugin/src/rules/only-throw-error.ts

```diff
@@ -39,6 +45,10 @@ export default createRule<Options, MessageIds>({
         type: 'object',
         additionalProperties: false,
         properties: {
+          allowThrowing: {
+            ...readonlynessOptionsSchema.properties.allow,
```

Comment:
[Refactor] The actual value for `.allow` is the `typeOrValueSpecifiersSchema` value from type utils:

```ts
import { typeMatchesSomeSpecifier } from '../utils';
```

```suggestion
            ...typeOrValueSpecifiersSchema,
```